### PR TITLE
Xpanse api updates

### PR DESCRIPTION
**Pull request by andrew-paloalto**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-5829

## Description
Several updates to the incoming mapper and field association. Also set a default classifier/mapper for the Cortex Xpanse integration.

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CortexXpanse/IncidentFields/incidentfields-Xpanse_Provider.json
Comment (johnnywilkes) [inline Packs/CortexXpanse/IncidentFields/incidentfields-Xpanse_Provider.json:22]:
```suggestion
    "associatedToAll": false,
    "associatedTypes": [
        "Xpanse Alert"
    ],
```

**Review by johnnywilkes — APPROVED** [review]
@andrew-paloalto , approved

**Comment by johnnywilkes** [comment]
@melamedbn  / @MichaelYochpaz , this is approved on Xpanse side and I don't think we cause the build errors.  Please merge when possible.

**Review by MichaelYochpaz — APPROVED** [review]
Looks good :)
Just waiting for @melamedbn's approval.

**Pull request merged**
